opsdroid/events.py

```diff
@@ -271,6 +270,26 @@ def __repr__(self):
         return await super().respond(response)
 
 
+class EditedMessage(Message):
+    """A  `opsdroid.events.Message` which has been edited.
+
+    The ``linked_event`` property should hold either an `opsdroid.events.Event`
+    class or an id for an event to which the edit applies.
+    """
+
+    def __init__(self, *args, **kwargs):
+        """Create object with minimum properties."""
+        super().__init__(*args, **kwargs)
```

Comment:
`linked_event` is already an attribute that you can set on the base `Message` class then?

Reply:
it's on `Event`.